content/en/docs/reference/kubernetes-api/labels-annotations-taints.md

```diff
@@ -114,3 +116,151 @@ The scheduler (through the _VolumeZonePredicate_ predicate) also will ensure tha
 If `PersistentVolumeLabel` does not support automatic labeling of your PersistentVolumes, you should consider
 adding the labels manually (or adding support for `PersistentVolumeLabel`). With `PersistentVolumeLabel`, the scheduler prevents Pods from mounting volumes in a different zone. If your infrastructure doesn't have this constraint, you don't need to add the zone labels to the volumes at all.
 
+## node.kubernetes.io/windows-build {#nodekubernetesiowindows-build}
+
+Example: `node.kubernetes.io/windows-build=10.0.17763`
+
+Used on: Node
+
+Starting in v1.17, this label is automatically added to adapt to the situation that the same cluster uses multiple versions of windows server.
```

Comment:
```suggestion
When the kubelet is running on Microsoft Windows, the kubelet automatically labels its node
to record the version of Windows Server in use.
```

Mentioning v1.17 isn't needed; this PR targets documentation for the current (v1.20) release.